backend/rs/ic_vetkeys/src/utils/mod.rs

```diff
@@ -535,29 +619,20 @@ impl IBECiphertext {
     /// To decrypt this message requires using the VetKey associated with the
     /// provided derived public key (ie the same master key and context string),
     /// and with an `input` equal to the provided `identity` parameter.
-    pub fn encrypt(
-        dpk: &DerivedPublicKey,
-        identity: &[u8],
-        msg: &[u8],
-        seed: &[u8],
-    ) -> Result<IBECiphertext, String> {
-        let seed: &[u8; IBE_SEED_BYTES] = seed
-            .try_into()
-            .map_err(|_e| format!("Provided seed must be {} bytes long ", IBE_SEED_BYTES))?;
```

Comment:
We can remove the respective comment ("seed must be exactly 256 bits") in the docs.

Reply:
Fixed